### ISSUE TITLE
Validations not working on long user inputs

### ISSUE THREAD

**Issue by aortiz1** [microsoft/botframework-sdk#6336]
## Describe the bug

On long user input the bot responds with an error "Response status code does not indicate success: 400 (Bad Request).", validations for the input length don't prevent this error to show.

## Version

Composer  1.4.1
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser

- [* ] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [ *] Windows
- [ ] Ubuntu

## To Reproduce

1. Create a bot application and add a text input.
2. On the validatation tab add the condition "length(string(this.value))<255" and as a response on invalid prompt "the text is too long".
3. Add a trigger with some some phrases ie. "the machine is broken"
4. Put the following text on the user input "Lorem ipsum dolor sit amet, consectetuer adipiscing elit. Aenean commodo ligula eget dolor. Aenean massa. Cum sociis natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Donec quam felis, ultricies nec, pellentesque eu, pretium quis, sem. Nulla consequat massa quis enim. Donec pede justo, fringilla vel, aliquet nec, vulputate eget, arcu. In enim justo, rhoncus ut, imperdiet a, venenatis vitae, justo. Nullam dictum felis eu pede mollis pretium. Integer tincidunt. Cras dapibus. Vivamus elementum semper nisi. Aenean vulputate".
5. Bot responds "Response status code does not indicate success: 400 (Bad Request)."

## Expected behavior

After giving the input the bot should respond "the text is too long"

## Screenshots

![image](https://user-images.githubusercontent.com/30127751/123707974-37bb8c80-d830-11eb-9af4-ce9b0d3ff5ab.png)
![image](https://user-images.githubusercontent.com/30127751/123708018-502ba700-d830-11eb-98e7-ca91529b411c.png)

## Additional context

The issue only affects dialogs with  LUIS recognition triggers

**Comment by dmvtech** [comment]
@srinaath Good point. I checked with NodeJS runtime and it doesn't reproduce. I'll move to dotnet.
